go.mod

```diff
@@ -1,50 +1,17 @@
 module github.com/getsentry/sentry-go
 
+go 1.13
```

Comment:
Yes, it will work with >=go1.13 as explained above, you can add build tags like `// +build go1.13` to the new iris files and others like `// +build go1.11` to support older versions of it as well.

Reply:
@kataras the build tags are fine :+1: . This comment is about the `go` directive in the `go.mod` file :wink: 

The bump from 1.11 to 1.13 in this PR could lead to confusing output for users of sentry-go building their code with the still supported versions Go 1.11 or 1.12.

Tracking as #83.